<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 231);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>지역 이름 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_area">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="weather_btn">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 255, 248);</string>
    </property>
    <property name="text">
     <string>날씨 조회</string>
    </property>
   </widget>
   <widget class="QLabel" name="area_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(162, 168, 255)</string>
    </property>
    <property name="text">
     <string>지역 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temper_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>온도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="yesterday_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>어제와의 날씨비교</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust2_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>397</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1.2px solid rgb(175, 175, 175);
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 190, 184);
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>초미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="weather_img_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>날씨</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 252, 198);
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>미세먼지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 255, 254);
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>체감온도</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dust1_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>397</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1.2px solid rgb(175, 175, 175);
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sense_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>397</y>
      <width>70</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕 Semilight</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1.2px solid rgb(175, 175, 175);
border-radius:7px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>406</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
